src/sources/Mvt.js

```diff
@@ -24,14 +24,41 @@ const MVT_TO_CARTO_TYPES = {
     3: geometryTypes.POLYGON
 };
 
+/**
+ * A MVTOptions object declares a MVT configuration
+ * @typedef {object} MVTOptions
+ * @property {string} layerID - layerID on the MVT tiles to decode, the parameter is optional if the MVT tiles only contains one layer
+ * @property {function} viewportZoomToSourceZoom - function to transform the viewport zoom into a zoom value to replace `{z}` in the MVT URL template, undefined defaults to `Math.ceil`
```

Comment:
@property {function} [viewportZoomToSourceZoom=Math.ceil]  [jsDoc default value](http://usejsdoc.org/tags-param.html#optional-parameters-and-default-values)

Reply:
changed